indico/modules/events/registration/schemas.py

```diff
@@ -0,0 +1,20 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2020 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+from marshmallow.fields import String
+
+from indico.core.marshmallow import mm
+
+
+class RegistrationFormSchema(mm.ModelSchema):
+    class Meta:
+        from indico.modules.events.registration.models.forms import RegistrationForm
```

Comment:
I don't think having the import here has any benefit. This still runs at import time...

Reply:
This is the fix for [this](https://github.com/indico/indico/pull/4532/checks?check_run_id=870558678). I previously thought it wasn’t working but I mistakenly imported it incorrectly.

Reply:
If this particular line fixes it I'm very curious how and why... Because the code here runs immediately:

```
>>> class Foo(object):
...     class Meta:
...             print 'hi'
...             x = 1
...
hi
```

And at the point where it runs, nothing is defined compared to what would happen if you did the import right before the class.